src/pages/Detail/ReviewList.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Review from './Review';
+
+function ReviewList({ reviewData }) {
+  return (
+    <div className="review">
+      {reviewData.map(review => {
+        return (
+          <Review
+            key={review.id}
+            email={review.email}
+            boughtProduct={review.boughtProduct}
+            quantity={review.quantity}
+            reviewRate={review.reviewRate}
+            createdAt={review.createdAt}
+            reviewComment={review.reviewComment}
+          />
```

Comment:
```suggestion
          <Review
            key={review.id}
            reviewData={review}
          />
```
이렇게 한 번에 보내준 후에 받는 쪽에서 구조분해할당으로 풀어서 사용하세요!